HIP/hip-1056.md

```diff
@@ -256,16 +323,16 @@ full complexity of the merkle state tree.
 The Block Stream will be defined by the continuous transmission of Block information defined as follows.
 - Each `Block` represents all inputs and outputs of the consensus network for zero or more whole `Rounds`. The maximum
   number of rounds per block will be configurable and start at 1 round per block. It could be increased later if
-  necessary for performance reasons or if round become very frequent and small.
+  necessary for performance reasons or if rounds become very frequent and small.
     - A block *may* be zero rounds because the last block prior to a network "freeze" must communicate the state hash
       (via a block proof) of the state immediately prior to network shutdown. This is only possible with an extra empty
       block because the state hash in each block proof is the state hash at the *beginning* of that block/round. This
       delay of state hash communication is required to simultaneously meet Hedera performance goals and also meet EVM
       expectations for immediate access to recent (the last 256) block hash values.
-- A `Round` is a sequence of zero or more `Events` that have reached consensus at the same time. Round length in real
+- A `Round` contains a sequence of zero or more `Events` that have reached consensus at the same time. Round length in real
   world time depends on internet latency and event creation rate. In Hedera mainnet today it is around 1 round per
   second. In the future this could be shorter to lower latency, but would never make sense to be less than ping time.
-- An `Event` is a sequence of zero or more `EventTransactions` submitted by a single node.
+- An `Event` contains a sequence of zero or more `EventTransactions` submitted by a single node.
 - Each `EventTransaction` is either a `SignedTransaction` or a system transaction.
 - A `SignedTransaction` is transmitted as a byte array, contains the signed transaction bytes of a user-submitted
```

Comment:
This has changes since this HIP was written. We are in the process of merging the only system transaction we have into the hedera-defined transaction structure. I believe all the transactions will be `SignedTransaction` and the say you differentiate a system vs user submitted transaction is looking at a flag in the TransactionBody or determining it based on the TransactionBody oneof type (StateSignatureTransaction will be one of these types)